spring-boot/admin/src/main/java/eu/coatrack/admin/DatabaseInitializer.java

```diff
@@ -395,7 +398,7 @@ private ApiKey createNewApiKey(
 
         apiKey = apiKeyRepository.save(apiKey);
 
-        serviceApi = serviceApiRepository.findOne(serviceApi.getId());
+        serviceApi = serviceApiRepository.findById(serviceApi.getId()).orElseThrow(NotFoundException::new);
```

Comment:
As this PR is about updating dependencies, we should - as far as possible - keep the existing functionality in line with the previous version. 

That being said: the previously used method `findOne` was specified/documented as: "Returns:
the entity with the given id or null if none found". So I would suggest:
```suggestion
        serviceApi = serviceApiRepository.findById(serviceApi.getId()).orElse(null);
```
Anyway, if in some situations "throwing an exception" would make more sense, we should do that in a separate issue that is not about updating dependencies.

Reply:
Done.